Blavaan/Blavaan Jags Regression.Rmd

```diff
@@ -20,6 +20,14 @@ In this tutorial, we start by using the default prior settings of the software.
 
 We are continuously improving the tutorials so let me know if you discover mistakes, or if you have additional resources I can refer to. The source code is available via [Github](https://github.com/Rensvandeschoot/Tutorials). If you want to be the first to be informed about updates, follow me on [Twitter](https://twitter.com/RensvdSchoot).
 
+### How to cite this tutorial in APA style
+
+Smeets, Laurent & Van de Schoot, Rens. (2019). Bayesian Regression in Blavaan (using Jags). Zenodo. https://doi.org/10.5281/zenodo.4050170
```

Comment:
Let me suggest changing the authors' names following the APA style.
``Smeets, L. &, Van de Schoot, R. (2019). Bayesian Regression in Blavaan (using Jags). Zenodo. https://doi.org/10.5281/zenodo.4050170``

Reply:
Ok, This is a good idea for future versions, but for now this is nor part of the Zenodo upload of this tutorial.